tests/CMakeLists.txt

```diff
@@ -12,19 +12,44 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+cmake_minimum_required(VERSION 3.9)
+project(clvk_test)
```

Comment:
You might want CI coverage for a standalone build of the tests. BTW, what's the use-case?

Reply:
In ChromeOS build system, the OpenCL library and the tests are build separately.

Reply:
I tried to add a test in the CI, but I have never use azure...

Reply:
Ok, makes sense. Right, don't burn time on this then. I want to move to Github Actions ultimately (happy to get help there if you can/want).